### PR TITLE
Meeting Logs Kovri PR (2018-05-28) & Kovri DefCon (2018-05-29)

### PR DESCRIPTION
**Pull request by selsta**
Not sure about summary, tags and author.

### CODE REVIEW

**Comment by anonimal** [comment]
Thanks for doing this, @selsta.

File: _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md [hunk 1]
```diff
@@ -0,0 +1,447 @@
+---
+layout: post
+title: Overview and Logs for the Kovri PR Meeting Held on 2018-05-28
+summary: Discussion of Kovri public relations
+tags: [dev diaries, i2p, crypto]
+author: selsta / fluffypony
+---
+
```
Comment (el00ruobuob) [inline _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md]: Could you add the date here, as on all the posts? `*May 28th, 2018*`

**Comment by el00ruobuob** [comment]
Regarding the summary, as there was no official schedule and topics, it seems accurate to the discussion.
Regarding the tags, they seems logical for a Kovri meeting.
Regarding the author, I can't see fluffypony here. Maybe he's name is not mandatory? Why not go with `selsta / anonimal` instead?

**Comment by anonimal** [comment]
Referencing https://github.com/monero-project/meta/issues/225.

>Why not go with selsta / anonimal instead?

Yes, that would make more sense.

**Comment by el00ruobuob** [comment]
I just edit my suggestion to reflect the reality of the most active participant in the meeting.

File: _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md (hunk 1 above)
Comment (selsta) [reply _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md]: I did it like the other log posts, they don’t have a date too.

**Comment by selsta** [comment]
Updated the author :)

File: _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md (hunk 1 above)
Comment (el00ruobuob) [reply _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md]: There are, but only for September and October 2017... i looked at the latest kovri one to see this. ATM, it is not a big deal, but maybe we should update the lot to be consistent.

**Comment by el00ruobuob** [comment]
Tested ok on the following platforms:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

File: _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md (hunk 1 above)
Comment (el00ruobuob) [reply _posts/2018-05-28-logs-for-the-kovri-pr-meeting-held-on-2018-05-28.md]: It's definitely something to remove where it's present:
image

**Comment by dEBRUYNE-1** [comment]
@selsta - any `_` without `\` will break the formatting on the website. Thus, either remove all `_` from the logs or add a `\` in front of it.

**Comment by el00ruobuob** [comment]
I didn't notice it break anything. Sorry about that

**Comment by selsta** [comment]
Thanks @dEBRUYNE-1 and @el00ruobuob 

Escaped all the `_`

**Pull request merged**